### DISCUSSION TITLE
Sonoff 4CH Pro Update

### DISCUSSION THREAD

**Discussion by nasked** [arendst/Tasmota#11552] [Q&A] [answered]
I'm updating my Sonoff 4CH Pro to 9.3.1 Version with tasmota-minimal.bin and show me Version not compatible.
I tried with 9.2.0 with same result.
I tried to upgrade with tasmota.bin and show mi Upload buffer miscompare

Actually i running V.9.1.0

**Comment by sfromis** [comment, marked as answer]
Always use the compressed .gz binaries for OTA upgrading. This should get around this issue.
